### ISSUE TITLE
Contradicting Documentation

### ISSUE THREAD

**Issue by sudeepg100** [MicrosoftDocs/OfficeDocs-SharePoint#248]
Under the recommended architecture for host named site collections, the guidance is to include personal sites in the same web application as others. But this is not the correct guidance. The product itself includes a rule that checks if the personal sites are on a dedicated web application. See this

https://docs.microsoft.com/en-us/sharepoint/technical-reference/validate-the-my-site-host-and-individual-my-sites-are-on-a-dedicated-web-applica

So this guidance must be corrected. Logically including personal sites in same DB as other sites makes no sense. The correct guidance must indicate that one web application pool will be used (so as to not exhaust the limit of application pools per server). Two web applications can share the same application pool. Host header site collections must be in one web application and my sites should be in other web application.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4a3e7eee-a034-87ea-4c52-f31c89d7aafd
* Version Independent ID: c6c07b34-be7a-7d90-a898-0db8dd641f04
* Content: [Host-named site collection architecture and deployment in SharePoint Server](https://docs.microsoft.com/en-us/sharepoint/administration/host-named-site-collection-architecture-and-deployment)
* Content Source: [SharePoint/SharePointServer/administration/host-named-site-collection-architecture-and-deployment.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/SharePoint/SharePointServer/administration/host-named-site-collection-architecture-and-deployment.md)
* Product: **sharepoint-server-itpro**
* GitHub Login: @Techwriter40
* Microsoft Alias: **kirks**

**Comment by AndreaBarr** [comment]
@sudeepg100 Thank you for this feedback

I will get this over to the SharePoint team for investigation and updating. Thank you for taking the time to make the docs better. 

I was able to get this into the backlog for the SharePoint team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.
